ui/appui-react/src/appui-react/layout/widget-panels/Grip.scss

```diff
@@ -18,6 +18,8 @@ $line-grip-width: calc(#{$nz-grip-width} - 2px);
     position: relative;
     display: flex;
     transition: 0.2s ease-out;
+    user-select: none;
```

Comment:
Instead of adding `user-select: none` in all places, another solution would be to temporarily add it only on `<body>` (using JS) for the duration of dragging. This would also allow the user to select the text when they're not dragging.